index.html

```diff
@@ -171,42 +173,52 @@ <h1>ODA Tooling</h1>
       <div class="container marketing">
 
         <!-- Three columns of text below the carousel -->
-        <div class="row">
-          <div class="col-lg-3 text-center">
+        <div class="row justify-content-md-center" >
```

Comment:
This changes are not needed here, just use `col-lg-XXX` in all places and make sure that the name is short of title
```suggestion
        <div class="row" >
```